srcpkgs/ImageMagick/template

```diff
@@ -1,7 +1,9 @@
 # Template file for 'ImageMagick'
+_release_prefix=44
 pkgname=ImageMagick
-version=7.1.0.43
+version=7.1.0.44
```

Comment:
```suggestion
version=7.1.0.44
_upstream_version="${version%.${_release_prefix}}-${_release_prefix}"
```
and then replace it where it's used, perhaps

Reply:
Done